cfme/tests/ansible/test_embedded_ansible_rest.py

```diff
@@ -100,11 +100,7 @@ def test_delete_repository_from_detail(self, appliance, repository, method):
         Metadata:
             test_flag: rest
         """
-        if method == 'post':
-            del_action = repository.action.delete.POST
-        else:
-            del_action = repository.action.delete.DELETE
-
+        del_action = getattr(repository.action.delete, method.upper())
```

Comment:
QUESTION: is it possible to use `delete_resources_from_detail` here?

Reply:
@otsuman Not at this moment, there are BZs related to this response that we need to check (see in the code below).